pkg/reconciler/pipeline/controller.go

```diff
@@ -448,6 +448,7 @@ func (r *pipelineReconciler) createOrUpdateDaemonDeployment(ctx context.Context,
 	log := logging.FromContext(ctx)
 	isbSvcType, envs := sharedutil.GetIsbSvcEnvVars(isbSvcConfig)
 	envs = append(envs, corev1.EnvVar{Name: dfv1.EnvPipelineName, Value: pl.Name})
+	envs = append(envs, corev1.EnvVar{Name: dfv1.EnvGrpcAlpnEnabled, Value: "false"})
```

Comment:
Why don't go with the approach of `setenv` in the source code? That will be easier to clean up in the future.

Reply:
https://github.com/grpc/grpc-go/blob/b368379ef8f6fbe8b3b856d38dbb34a50fbfbe9a/internal/envconfig/envconfig.go#L48
It is fetched during init

Reply:
Should it be okay if we add it in the place of code start? Adding it to the manifests, especially the server and controller manifests is nasty, and not able to clean up if ppl do `kubectl apply` during upgrade.